internal/weldr/api.go

```diff
@@ -465,17 +469,26 @@ func (api *API) getSourceConfigs(params httprouter.Params) (map[string]store.Sou
 	sources := map[string]store.SourceConfig{}
 	errors := []responseError{}
 
+	repos, err := api.repoRegistry.ReposByArch(api.arch, false)
```

Comment:
Also here, should the user be able to get config of a image-type-specific repo/source? Currently they won't be able to do so.